assets/src/wizards/subscriptions/index.js

```diff
@@ -31,43 +34,83 @@ class SubscriptionsWizard extends Component {
 	constructor() {
 		super( ...arguments );
 		this.state = {
+			currentTask: 0,
 			inputTextValue1: "Input value",
 			inputTextValue2: "",
 			image: null,
+			completedTasks: [ 2 ],
 			selectValue1: "2nd",
 			selectValue2: "",
 		}
 	}
 
 	/**
-	 * Render the example stub.
+	 * componentDidMount.
+	 */
+	componentDidMount() {
+		this.setState( { currentTask: this.getNextTask( 0, this.state.completedTasks) } )
+	}
```

Comment:
- just to keep things clean, extracted this into a separate function:

```suggestion
	}

	/**
	 * Starting from the startingIndex, determine the next task to go to.
	 *
	 * @param int startingIndex
	 * @param Array completedTasks
	 * @returns int {number}
	 */
	getNextTask = ( startingIndex, completedTasks ) => {
		let nextTaskIndex = 0;
		while ( completedTasks.indexOf( nextTaskIndex ) !== -1 ) {
			nextTaskIndex++;
		}

		return nextTaskIndex;
	}
```

Reply:
**This conversation/segment is a dupe, please ignore.**